schema/material/conventional.json

```diff
@@ -0,0 +1,10 @@
+{
+    "schemaId": "material-conventional",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "material conventional schema",
+    "properties": {
+        "conventional": {
```

Comment:
I understand that this schema had to be made to replicate `BankMaterialsSchema` in the web-app. However, for ESSE it doesn't make all too much sense (very simple object that is not being reused). We should probably revisit this in the future.

Reply:
What is stored inside the `conventional` flag - is it another material structure? If yes, should we refer to it for clarity, and not just "object"?

Reply:
As discussed with @spectre007 in person - let's have a followup task to re-organize the schemas and move some out of `system`. This work effort is aimed at making the webapp use ESSE schemas instead of duplicating them.